src/definitions/modules/dropdown.js

```diff
@@ -4093,6 +4099,12 @@ $.fn.dropdown.settings.templates = {
             : ''
         ;
         html += '<div class="'+ maybeDisabled + className.item+'" data-value="' + String(option[fields.value]).replace(/"/g,"") + '"' + maybeText + '>';
+        if(option[fields.image]) {
+          html += '<img class="'+(option[fields.imageClass] ? String(option[fields.imageClass]).replace(/"/g,"") : className.image)+'" src="' + String(option[fields.image]).replace(/"/g,"") + '">';
```

Comment:
Repeated `String(option[...]).replace(/"/g,"")` should be extracted into a function.

Reply:
Done by 1678852d86bf103a0149623715b23c3f18c7818c 🙂